tensorflow/python/kernel_tests/manip_ops_test.py

```diff
@@ -98,41 +100,68 @@ def testNegativeAxis(self):
         manip_ops.roll(np.random.randint(-100, 100, (4, 4)).astype(np.int32),
                        3, -10).eval()
 
+  def testInvalidInputShape(self):
+    # The input should be 1-D or higher, checked in shape function.
+    with self.assertRaisesRegexp(
+        ValueError, "Shape must be at least rank 1 but is rank 0"):
+      manip_ops.roll(7, 1, 0)
+
   def testRollInputMustVectorHigherRaises(self):
-    tensor = 7
+    # The input should be 1-D or higher, checked is done in kernel.
```

Comment:
minor nits: just for consistency, maybe change to checked in kernel.

Reply:
@yzhwang The comment has been updated. Thanks for the review!